diff --git a/cjs/react-dom-client.production.js b/cjs/react-dom-client.production.js
index a2b010df6add576b3a364997c90fc0db841fed1c..8e442c8b94cdb523edbf0ba8085e63dcdab3bf50 100644
--- a/cjs/react-dom-client.production.js
+++ b/cjs/react-dom-client.production.js
@@ -14520,8 +14520,9 @@ function setProp(domElement, tag, key, value, props, prevValue) {
         key = value.__html;
         if (null != key) {
           if (null != props.children) throw Error(formatProdErrorMessage(60));
-          (null != prevValue ? prevValue.__html : void 0) !== key &&
-            (domElement.innerHTML = key);
+          throw Error(
+            "dangerouslySetInnerHTML is disabled (patched out for AMO review)."
+          );
         }
       }
       break;
@@ -14738,8 +14739,9 @@ function setPropOnCustomElement(domElement, tag, key, value, props, prevValue) {
         key = value.__html;
         if (null != key) {
           if (null != props.children) throw Error(formatProdErrorMessage(60));
-          (null != prevValue ? prevValue.__html : void 0) !== key &&
-            (domElement.innerHTML = key);
+          throw Error(
+            "dangerouslySetInnerHTML is disabled (patched out for AMO review)."
+          );
         }
       }
       break;
